apikey-service/src/main/java/eu/europeana/apikey/controller/ApikeyController.java

```diff
@@ -658,6 +675,15 @@ private void mandatoryMissing(ApikeyAction apikeyUpdate) throws ApikeyException
             throw new ApikeyException(400, BAD_EMAIL_FORMAT, BAD_EMAIL_FORMAT);
         }
     }
-
+    private void apiKeyAlreadyExist(ApikeyAction apikeyUpdate) throws ApikeyException{
+        Apikey apikey= this.apikeyRepo.findByEmailAndAppName(apikeyUpdate.getEmail(), apikeyUpdate.getAppName());
+        System.out.println(apikey);
```

Comment:
Please do not use System.out or System.err (unless it's temporary for debugging). For permanent code use LOG.debug instead or LOG.info instead.